### PR TITLE
Redesign of system admin pages

### PR DESCRIPTION
**Pull request by alvak1**
Minor changes to components of system admin pages listed below

- Paragraph added to manage organisation page
- Paragraph added to add organisation page
- Status column added to table in add organisation page
- Changes to data displayed for sys admin in account settings page
- Rows sorted according to name in manage organisation page

### CODE REVIEW

File: site/pages/sysadmin/manage-organisations.page.tsx [hunk 1]
```diff
@@ -79,11 +79,13 @@ const ManageOrganisations = ({ orgList, csrfToken }: ManageOrganisationsProps):
                     ]}
                 />
             ) : null}
-            <Table
-                caption={{ text: "Manage organisations", size: "m" }}
```
Comment (ljones92) [inline site/pages/sysadmin/manage-organisations.page.tsx:83]: Looking at the designs I think this should be "govuk-body", the designs do have a lower font weight but I'd ignore that for now

File: site/pages/sysadmin/users.page.tsx [hunk 2]
```diff
@@ -138,7 +139,12 @@ const AdminUsers = (props: AdminUserProps): ReactElement => {
                 />
             ) : null}
             <ErrorSummary errors={pageState.errors} />
-            <h1 className="govuk-heading-xl">Add new user</h1>
+            <h1 className="govuk-heading-xl">Add an organisation admin</h1>
+            <p className="govuk-body-s">
```
Comment (ljones92) [inline site/pages/sysadmin/users.page.tsx:143]: Same here

File: site/pages/sysadmin/manage-organisations.page.tsx (hunk 1 above)
Comment (alvak1) [reply site/pages/sysadmin/manage-organisations.page.tsx:83]: Are you referring to the heading "Manage organisations"? i have updated it "govuk-heading-l" as this is used in most of the pages for a heading name.

File: site/pages/account-settings.page.tsx [hunk 3]
```diff
@@ -68,28 +69,62 @@ const AccountSettings = ({ sessionWithOrg, csrfToken }: AccountSettingsProps): R
                 <h2 className="govuk-heading-m">Account settings</h2>
```
[the remark this answers is not in the document]
Comment (alvak1) [reply site/pages/account-settings.page.tsx]: done

File: site/pages/sysadmin/users.page.tsx (hunk 2 above)
Comment (alvak1) [reply site/pages/sysadmin/users.page.tsx]: done

File: site/pages/sysadmin/manage-organisations.page.tsx (hunk 1 above)
Comment (ljones92) [reply site/pages/sysadmin/manage-organisations.page.tsx:83]: Sorry yeah, commented in the wrong place

File: site/pages/sysadmin/manage-organisations.page.tsx [hunk 4]
```diff
@@ -79,11 +79,13 @@ const ManageOrganisations = ({ orgList, csrfToken }: ManageOrganisationsProps):
...
-                columns={["Organisation", "NaPTAN AdminArea", "Action"]}
-                rows={getRows()}
-            ></Table>
+            <h1 className="govuk-heading-l">Manage organisations</h1>
+            <p className="govuk-body">
+                Create, edit and remove organisations who can use the CTDD tool. Assign NaPTAN AdminArea&apos;s per
```
Comment (ljones92) [inline site/pages/sysadmin/manage-organisations.page.tsx:84]: I know it's in the designs but there shouldn't be an apostrophe here
```suggestion
                Create, edit and remove organisations who can use the CTDD tool. Assign NaPTAN AdminAreas per
```

Comment (alvak1) [reply site/pages/sysadmin/manage-organisations.page.tsx]: done

**Review by ljones92 — APPROVED** [review]

**Pull request merged**